components/file-uploader/render-state.tsx

```
@@ -49,14 +70,57 @@ export function RenderUploadedState({
+                    <p className="text-muted-foreground text-xs uppercase">{extension} file</p>
+                </div>
+            </div>
+        );
+    };
+
     return (
-        <div>
-            <Image src={previewUrl} alt="uploaded file" fill className="object-contain p-2" />
+        <div className="group relative flex h-full w-full items-center justify-center">
+            {fileType === 'video' ? (
+                <video src={previewUrl} controls className="h-full w-full rounded-md" />
```

Biome (2.1.2)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)